portal/styles/button.scss

```diff
@@ -4,4 +4,8 @@
   text-decoration: underline;
   cursor: pointer;
   color: $govuk-link-colour;
+  padding: 0;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-family: GDS Transport, arial, sans-serif;
```

Comment:
I expected we'd want `font: inherit` here instead

I hope doing that also means we don't need to use `govuk-!-font-size-19` anymore either

Reply:
Great suggestion, I've made this change. You're absolutely right that this negates the need for `govuk-!-font-size-19` too